### PR TITLE
Adding Directories, Assets, Updated Contract & Polygon Network

### PR DESCRIPTION
**Pull request by myweb3journey**
1. Adding directories:
- credential-token_goerli: contract deployed to goerli network
- credential-token_polygon: contract deployed to mumbai network
- assets: JSON objects for our credential images

2. Updated Contract: 
- Contains the updated assets

3. Polygon Network: Contracts been deployed to polygon network using hardhat

### CODE REVIEW

File: assets/base.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "description": "Women In Web 3 - Credential Token",
```
Comment (seemcat) [inline assets/base.json:2]: **WOMEN IN WEB3** across all pages (everywhere) pls!

File: credential-token_goerli/contracts/WIW3CredentialToken.sol [hunk 2]
```diff
@@ -60,10 +60,10 @@ contract WIW3CredentialToken is ERC721, ERC721URIStorage, Ownable {
 
 	// Upon contract creation, set all Token URIs
 	constructor() ERC721("WOMEN IN WEB3 Credential Tokens", "CT") {
-		base = "https://womeninweb3.infura-ipfs.io/ipfs/QmWNd3c4cD4BngaxV3xHbZM6ewTz1Mh8V3BAfdUXtCECqP";
-		learn = "https://womeninweb3.infura-ipfs.io/ipfs/QmdcVTYPtB3vPjWoVvsdRte4EAAFW3zApBK4pugaAMeyi7";
-		prototype = "https://womeninweb3.infura-ipfs.io/ipfs/QmU5acPa8M4SG2r4t8X9Uz6eBovpiUf3tiR63Uv4SnmF7o";
-		venture = "https://womeninweb3.infura-ipfs.io/ipfs/QmXzLByLGqXH528r5iWxqEf8EZuY6Y6m1NMWLRhh6BLH2G";
+		base = "https://ipfs.io/ipfs/QmVk3LQxKt8kpJNn8XKaa6DUQ7ztHmvHXusuJuypqVSV5W?filename=base.json" ;
+        learn = "https://ipfs.io/ipfs/QmXuBH1sBZobCbk7aenKv7dowbQi4Z8C1vCrLHgm2Jt7HL?filename=learn.json";
```
Comment (seemcat) [inline credential-token_goerli/contracts/WIW3CredentialToken.sol:64]: Can we pls use Prettier for Solidity to ensure indentation is correct?

File: credential-token_polygon/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# The WOMEN IN WEB3 Member Portal
+A token-gated website for WIW3 members to go through the Venture Builder and earn credential tokens as they complete specific milestones.
+
+## Getting Started
+
+### Part 1: Alchemy, Crypto Wallet & Install
```
Comment (seemcat) [inline credential-token_polygon/README.md:6]: Is there a reason why we need two different README files? Wondering if we can condense it into one?

File: credential-token_polygon/contracts/WIW3CredentialToken.sol [hunk 4]
```diff
@@ -0,0 +1,295 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
```
Comment (seemcat) [inline credential-token_polygon/contracts/WIW3CredentialToken.sol:2]: Can we please follow this framework for naming files? e.g. `credential-token_goerli` changes to `credential-token-goerli`

File: credential-token_polygon/contracts/WIW3CredentialToken.sol [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+	string public base;
+	string public learn;
+	string public prototype;
+	string public venture;
+
+	/* 🤔 How come we don't specify an owner of the credential token? */
+
+	// Upon contract creation, set all Token URIs
+	constructor() ERC721("WOMEN IN WEB3 Credential Tokens", "CT") {
+		base = "https://ipfs.io/ipfs/QmVk3LQxKt8kpJNn8XKaa6DUQ7ztHmvHXusuJuypqVSV5W?filename=base.json" ;
```
Comment (seemcat) [inline credential-token_polygon/contracts/WIW3CredentialToken.sol:63]: pls fix the indentation on the page using prettier!

**Review by seemcat — COMMENTED** [review]
Looks good Nima, TY for pushing. Just had minor feedback & Qs!

File: credential-token_goerli/contracts/WIW3CredentialToken.sol (hunk 2 above)
Comment (myweb3journey) [reply credential-token_goerli/contracts/WIW3CredentialToken.sol:64]: Sounds good!

File: assets/base.json (hunk 1 above)
Comment (myweb3journey) [reply assets/base.json:2]: Got it!

File: credential-token_polygon/README.md (hunk 3 above)
Comment (myweb3journey) [reply credential-token_polygon/README.md:6]: I guess we don't really need it, I added two different READMEs for each directory level of the project

**Pull request closed without merging**